cli.go

```diff
@@ -1094,6 +1110,22 @@ func actionInspect(c *cli.Context) error {
 	if pub {
 		pubObjs(topub, cl, c)
 	}
+	if qr {
+		// Generate a QR code for each entity with an available signing key
+		for _, qrd := range(toqrg) {
+			roi := qrd.ro
+			if (roi.GetRONum() == objects.ROEntity) {
+				entity := roi.(*objects.Entity)
+				signingblob := entity.GetSigningBlob()
+				if signingblob != nil {
+					err := qrcode.WriteFile(string([]byte{objects.ROEntityWKey}) + string(signingblob), qrcode.Medium, 256, fmt.Sprintf("%s.png", qrd.name))
```

Comment:
I would default to more than 256 pixels, or perhaps choose the resolution based on the size of signingblob